test/test_tree_2d_euler.jl

```diff
@@ -152,7 +157,8 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_blob_amr.jl"),
       l2   = [0.2086261501910662, 1.2118352377894666, 0.10255333189606497, 5.296238138639236],
       linf = [14.829071984498198, 74.12967742435727, 6.863554388300223, 303.58813147491134],
-      tspan = (0.0, 0.12))
+      tspan = (0.0, 0.12),
+      coverage_override = (maxiters=10^5,))
```

Comment:
Why do you use a big `maxiters` here?

Reply:
Same as above. I think I should add all these comments here as comments to the code.

Reply:
That would be great 👍